### DISCUSSION TITLE
Migrating keygen users from one account to another

### DISCUSSION THREAD

**Discussion by asaf-b** [keygen-sh/community#24] [Q&A] [answered]
Hi Zeke,
I'm currently working on a system upgrade to my licensing app.
In order to make sure I don't pollute the current keygen account, I've decided to open another account and start clean. 
At some point, I'll need to make a decision on which one to keep (maybe both for a while to support older product versions) after the transition- that will of course require a migration of some kind.
My question is- in what way would you suggest migrating users from one account to another?
In case I want to keep the new account, how can I migrate my users to the new DB? 
I'm asking because I obviously have no access to user passwords. So I can loop through and copy my users using the API, but how will I migrate their passwords as well?

Thank you,
Asaf Ben Zur

**Comment by ezekg** [comment, marked as answer]
The best way to do this would be to create a matching passwordless user on the new Keygen account, and then issue a password reset for each user. They can follow the link in the email to set a password for their user profile.

**Reply by asaf-b** [reply]
Amazing, I wasn't aware you can create a passwordless user, seen it now on the docs.
Thanks for the super fast reply!

**Reply by ezekg** [reply]
No worries — it's actually a new feature, released about a week ago! Works great for use cases like this.
